ui/raidboss/data/02-arr/raid/t7.js

```diff
@@ -82,12 +82,12 @@
     },
     {
       id: 'T7 Voice',
```

Comment:
Can this be T7 Cursed Voice? I know it's kind of pedantic, but we lose some context by moving to the effectId

Reply:
And `T7 Cursed Shriek` elsewhere too!

Reply:
This was done to resolve this issue. #1425 
As I did not understand the meaning of @panicstevenson  what should I do? @quisquous

Reply:
What they are saying is that "voice" as a trigger ID in English does not really explain what is going on with this trigger. Instead, it should be:

```
 {
      id: 'T7 Cursed Voice',
      // trigger body etc
  }
```

This way, we know what the mechanic is without having to look for clues or looking it up on XIVAPI.

(This is not your fault, the trigger was named this a very long time ago when we didn't have as much standardization for things. However, now that we are using bare effect IDs, we do need to have the name for it *somewhere* in the trigger.)